source/sass/global/variables.scss

```
@@ -0,0 +1,18 @@
+// color
+
+$color-green-1000: #68b738;
+$color-black: #000000;
+$color-grey-100: #e6e6e6;
+$color-grey-150: #e7e7e7;
+$color-grey-200: #f2f2f2;
+$color-grey-300: #d9d9d9;
+$color-grey-400: #eaeaea;
+$color-grey-600: #444444;
+$color-grey-700: #222222;
+$color-grey-900: #111111;
+$color-white: #ffffff;
```

По-моему, было бы логичнее наоборот:
```suggestion
$color-white: #ffffff;
$color-grey-100: #e6e6e6;
$color-grey-150: #e7e7e7;
$color-grey-200: #f2f2f2;
$color-grey-300: #d9d9d9;
$color-grey-400: #eaeaea;
$color-grey-600: #444444;
$color-grey-700: #222222;
$color-grey-900: #111111;
$color-black: #000000;
```

---

Поменяла